### ISSUE TITLE
MSG_BOLD missing in messages.prf

### ISSUE THREAD

**Issue by angband-import** [angband/angband#3858]
**Reported by draconisPW on 4 Nov 2013 10:14 UTC**
Currently, messages.prf and MSG_XXX enum are out of sync due to missing MSG_BOLD in messages.prf:

```
enum
{
...
MSG_BERSERK       = 64,
MSG_BOLD          = 65,
MSG_PROT_EVIL     = 66,
...
MSG_MAX           = 151
};
```

```
...

# MSG_BERSERK
M:64:w

# MSG_PROT_EVIL
M:65:w

...

# MSG_MAX 150
```

**Comment by angband-import** [comment]
**Modified by takkaria on 5 Nov 2013 23:37 UTC**

**Comment by angband-import** [comment]
**Modified by bensemmler on 12 Nov 2013 18:26 UTC**

**Comment by angband-import** [comment]
**Comment by takkaria on 25 Nov 2013 22:49 UTC**
Fixed in 5139323 with https://github.com/angband/angband/pull/283
